task2/Rectangle.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using task2.Interfaces;
+
+namespace task2
+{
+    class Rectangle : IShape, IMovable
+    {
+        const string FirstCorner = "╔";
+        const string SecondCorner = "╗";
+        const string ThirdCorner = "╚";
+        const string FourthCorner = "╝";
+        const string HorizontalElement = "=";
+        const string VerticalElement = "║";
+        public void Draw(int w, int h)
+        {
+            Console.SetCursorPosition(w,h);
+            string s = FirstCorner;
+            string space = "";
+            string first_space = "";
+            for (int i = 0; i < 5; i++)
+            {
+                space += " ";
+                s += HorizontalElement;
+            }
+
+            for (int i = 0; i <w; i++)
+            {
+  
+                first_space += " ";
+
+            }
+         
+            s += SecondCorner + "\n";
+
+            for (int i = 0; i < 5; i++)
+                s += first_space +VerticalElement + space + VerticalElement + "\n";
+
+            s +=  first_space + ThirdCorner;
+            for (int i = 0; i < 5; i++)
+                s += "═";
+
+            s += FourthCorner + "\n";
+
+
+            Console.Write(s);
+
+
+        }
+
+        public void Move()
```

Comment:
Logic will not work with arrow keys

Reply:
Move method we have in both classes but logic is the same. Put this in Base Shape class